rlberry/agents/agent.py

```diff
@@ -23,10 +23,10 @@
 class Agent(ABC):
     """Basic interface for agents.
 
-    If the 'inherited class' from Agent use the torch lib, that is higly recommanded to inherit :class:`~rlberry.agents.AgentTorch` instead.
+    If the 'inherited class' from Agent use the torch lib, that is highly recommended to inherit :class:`~rlberry.agents.AgentTorch` instead.
 
     .. note::
-        | 1 - Abstract Class : can't be cannot be instantiated. The abstract methods have to be overwriten by the 'inherited class' agent.
+        | 1 - Abstract Class : can't be cannot be instantiated. The abstract methods have to be override by the 'inherited class' agent.
```

Comment:
typo can't be cannot be instantiated -> cannot be instantiated.
The abstract methods have to be overridden by ...

Reply:
done !